app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
+    difficulty_easy: 0,
+    difficulty_some_what_easy: 1,
+    difficulty_ok: 2,
+    difficulty_some_what_difficult: 3,
+    difficult_too_difficult: 4,
+  }, _prefix: true
+
+  enum time: {
+    estimated_time_less_5: 0,
```

Comment:
Rubocop  agrees with the following names: 
`estimated_time_less5`, 
    `estimated_time5to10`,
    `estimated_time10to20`,
    `estimated_time20to30`,
    `estimated_time_more30`
    
Should I rename it that way and then adjust the names in the locales as well?

Reply:
Mh, I see. In this case, I actually don't agree with Rubocop, because I would rate the consistency more important. Let's disable rubocop with a magic comment:

```rb
# rubocop:disable Naming/VariableNumber
```

Later on, re-enable rubocop again:

```rb
# rubocop:enable Naming/VariableNumber
```

Reply:
As I see, adding a unit to those values also fixed the Rubocop annotation without disabling the `Naming/VariableNumber` rule 💪